src/containers/profile/profile.js

```diff
@@ -51,59 +54,55 @@ export const Profile = () => {
       <nav className={mobile_menu} style={menus}>
         <ul>
           <li>
-            <Link to="/profile/public">
+            <Link to="/profile-public">
               <button
-                type='button'
-                className={`profile_button ${getStatus === 0? 'profile_button_active': ''}`}
-                onClick={() => {
-                  toggleButton(0), setMenu({ marginLeft: '-105%' });
-                }}>
+                type="button"
+                className={`${button} ${
+                  currentTab === tabs.public ? active_button : ''
```

Comment:
I think that calculate component class is not looks so good for understanding code.
ClassNames package was recommended as a good way to prettify our code.
Please, Make me noted about your decision about that npm-package.